### PR TITLE
[bitnami/rabbitmq] fix: Typo in the port name of rabbitmq notes

### PR DESCRIPTION
**Pull request by QuentinN42**
As sayed in #10675 I've just fixed a small typo in the chart.

### CODE REVIEW

**Comment by QuentinN42** [comment]
Which version need I to upgrade ?

- `appVersion: 3.10.5` -> `appVersion: 3.10.6`
- `version: 10.1.5` -> `version: 10.1.6`

Sorry for the dumb question but I'm new to k8s and helm :smile:

**Comment by carrodher** [comment]
Thanks for your contribution, please check the failing actions; you need to bump the chart version

File: bitnami/rabbitmq/templates/NOTES.txt [hunk 1]
```diff
@@ -39,7 +39,7 @@ Note that the credentials are saved in persistent volume claims and will not be
 This is applicable when no passwords are set and therefore the random password is autogenerated. In case of using a fixed password, you should specify it when upgrading.
 More information about the credentials may be found at https://docs.bitnami.com/general/how-to/troubleshoot-helm-chart-issues/#credential-errors-while-upgrading-chart-releases.
 
-RabbitMQ can be accessed within the cluster on port {{ $serviceNodePort }} at {{ include "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
+RabbitMQ can be accessed within the cluster on port {{ $servicePort }} at {{ include "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
```
Comment (FraPazGal) [inline bitnami/rabbitmq/templates/NOTES.txt:42]: Could you please remove the `serviceNodePort` variable definition? It was only used here.

**Review by FraPazGal — COMMENTED** [review]
Thanks for the PR @QuentinN42, could you please look at my comment? Also, please check the PR conflict so we can merge it without issues.

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**